core/dictgen/src/rootcling_impl.cxx

```diff
@@ -3218,7 +3218,7 @@ class tempFileNamesCatalog {
    tempFileNamesCatalog(): m_size(0), m_emptyString("") {};
 
    std::string getTmpFileName(const std::string &filename) {
-      return filename + "_tmp";
+      return filename + "_tmp_" + std::to_string(getpid());
```

Comment:
Thanks, @Teemperor . Anything that prevents CMake from spawning two rootcling processes to build the same product is majorly appreciated by anyone building with `LLVM_BUILD_TYPE=Debug`! :-)